src/Vlingo.Actors/World.cs

```diff
@@ -425,7 +425,16 @@ public void RegisterDynamic(string name, object dep)
         /// <returns></returns>
         public TDependency ResolveDynamic<TDependency>(string name)
         {
-            return (TDependency)this.dynamicDependencies[name];
+            try
+            {
+                return (TDependency)this.dynamicDependencies[name];
+            }
+            catch (KeyNotFoundException)
+            {
+#pragma warning disable CS8653 // A default expression introduces a null value for a type parameter.
```

Comment:
I think that you can get rid of pragma by just doing `return default!` With nullable refereneces you can mark it this way

Reply:
K, good stuff.